sdb/commands/linux/per_cpu.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+import argparse
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class LxPerCpuPtr(sdb.SingleInputCommand):
+    """
+    Return the per-CPU pointer for the given CPU.
+
+    EXAMPLES
+        Get the per-CPU data of the "kmalloc-8" kmem cache for CPUs 0 and 1:
+
+            sdb> slabs | filter obj.name == "kmalloc-8" | member cpu_slab
+            (struct kmem_cache_cpu *)0x2f040
+            sdb> slabs | filter obj.name == "kmalloc-8" | member cpu_slab | cpu_ptr 0
+            (struct kmem_cache_cpu *)0xffff944effc2f040
+            sdb> slabs | filter obj.name == "kmalloc-8" | member cpu_slab | cpu_ptr 1
+            (struct kmem_cache_cpu *)0xffff944effd2f040
```

Comment:
What do you think about allowing the user to omit the cpu number, in which case the command will emit the pointer for each CPU that exists? That feels like an idiomatic way for the command to work. Also, I suspect that people will usually be less interested in the values on particular CPUs, and more interested in aggregating the values on all of the CPUs. Emitting a list of values would make it simple to do all sorts of aggregations.

Reply:
Changed it so without an arguments we print all CPUs but still allowing the user to specify an a specific CPU or set of CPUs if they so wish.